src/DistrictResources/DistrictResources.cpp

```diff
@@ -1,4 +1,250 @@
+                                                                                    workFactor)));
+                std::cout << "Adding vehicle #" << vehicleId << " to global list" <<std::endl;
+            }
+        }
+    }
+
+    void DistrictResources::processFireStationSampleData(const std::string &fileName)
+    {
+        std::vector<std::vector<std::string>> stations;
+        std::vector<std::string> row;
+        std::string line, word;
+
+        std::fstream file (fileName, std::ios::in);
+        while(getline(file, line))
+        {
+            row.clear();
+
+            std::stringstream str(line);
+
+            while(getline(str, word, ','))
+                row.push_back(word);
+            stations.push_back(row);
+        }
+
+        bool header = true;
+
+        for (std::vector<std::string> entry : stations)
+        {
+            if (header)
+            {
+                header = false;
+                continue;
+            }
+
+            auto const x = stof(entry.at(1));
+            auto const y = stof(entry.at(2));
+            auto const location = Location(x, y);
+            auto const stationId = stoi(entry.at(0));
+            auto const maxCrew = stoi(entry.at(3));
+            auto const curCrew = stoi(entry.at(4));
+
+            districtFireStations.push_back( std::make_unique<FireStation>( FireStation(stationId, location, maxCrew, curCrew) ));
+
+            std::cout << "Adding Fire Station #" << stationId << " to global list" << std::endl;
+        }
+    }
+
+    void DistrictResources::processEventSampleData(const std::string &fileName, std::vector<Event> *pendingQueue)
+    {
+        std::vector<std::vector<std::string>> events;
+        std::vector<std::string> row;
+        std::string line, word;
+
+        std::fstream file (fileName, std::ios::in);
+        while(getline(file, line))
+        {
+            row.clear();
+
+            std::stringstream str(line);
+
+            while(getline(str, word, ','))
+                row.push_back(word);
+            events.push_back(row);
+        }
+
+        bool header = true;
+
+        for (std::vector<std::string> entry : events)
+        {
+            if (header)
+            {
+                header = false;
+                continue;
+            }
+            std::string eventType = entry.at(3);
+            if (eventType == "CriticalSituation")
+            {
+                auto const eventID = stoi(entry.at(0));
+                auto const startTime = Time::stringToTime(entry.at(1));
+                auto const durationSeconds = stoi(entry.at(2));
+                auto const location = Location(stof(entry.at(4)), stof(entry.at(5)));
+                auto const vehicleRequirementsId = stoi(entry.at(6));
+                pendingQueue->push_back(Event(eventID, startTime, durationSeconds, location, vehicleRequirementsId));
+//
+                std::cout << "Adding Event #" << entry.at(0) << " to pending queue" << std::endl;
```

Comment:
```suggestion
#IFNDEF NDEBUG
                std::cout << "Adding Event #" << entry.at(0) << " to pending queue" << std::endl;
#ENDIF
```
Might be worth wrapping this with a compiler directive so that the logging only prints when you compile in debug mode (e.g., without `-O3`) and doesn't clutter `std::cout` in release mode.